docker-hub/access-tokens.md

```diff
@@ -28,15 +38,26 @@ Settings**.
 
 3. Select **Security > New Access Token**.
 
-      ![](images/hub-create-token.png)
-
 4. Add a description for your token. Use something that indicates where
 the token is going to be used, or set a purpose for the token.
 
-5. Copy the token that appears on the screen. Make sure you do this now:
+      ![PAT Menu](images/hub-create-token.png){: style="max-width: 60%"}
+
+5. You can view the following permissions from the drop-down:
+     * Read, Write, Delete: lets you to manage team members. This is an admin privilege.
+     * Read & Write: lets you to push images to any repository managed by your account.
+     * Read Only: lets you to view, search, and pull images from any public repositories and any private repositories that you have access to.
+     * Public Repo Read-only: lets you view, search, and pull images from any public repositories.
+
+ >**Note**
+ >
+ > All users have access to Read, Write, and Delete. Only paid users have
+ > access to Read & Write, Read-only, and Public Repo Read-only.
+
+6. Copy the token that appears on the screen. Make sure you do this now:
```

Comment:
This appears at list item 1 in the preview. Indenting the Note above should fix this issue

![image](https://user-images.githubusercontent.com/47779042/124263621-69cc3780-db2b-11eb-803c-0578cdceb262.png)

Reply:
@jerae-duffin This issue still exists in the preview - https://deploy-preview-13084--docsdocker.netlify.app/docker-hub/access-tokens/#create-an-access-token

Reply:
Updated